x-pack/test/functional/apps/maps/mapbox_styles.js

```diff
@@ -200,7 +200,7 @@ export const MAPBOX_STYLES = {
       'visibility': 'visible'
     },
     'paint': {
-      'line-color': '#FFFFFF',
+      'line-color': '#b8143c', // Derived color. The darkened version of #e6194b
```

Comment:
Stemming from my previous comment, this could also then get out of sync with the color palette. If you already have some JS function for darkening user-supplied colors, I'd use that here too and pull in the first color from the palette above.

Reply:
This is tricky. When the functional tests load, they haven't loaded the required resources to "see" eui import. Trying to import the default colors results in a build error due to missing eui lib dependencies. I've updated the tests to not test the color for now since it can/will change over time and is currently the only fluctuating style.

Reply:
If you need it just for tests, can you just apply a static hex in the test itself instead of removing the test altogether?